Scripts/script-InRange.yml

```diff
@@ -0,0 +1,51 @@
+commonfields:
+  id: InRange
+  version: -1
+name: InRange
+script: |
+  INVALID_ARG = "INVALID_ARG"
+  def to_float(s):
+      try:
+          return float(s)
+      except:
+          return INVALID_ARG
+
+  # "1,8" => will return a tupple (1, 8)
+  def parse_range(rangeStr):
+      splitted = rangeStr.split(",")
+      if len(splitted) < 2:
+          return INVALID_ARG, INVALID_ARG
+
+      # parse
+      return to_float(splitted[0]), to_float(splitted[1])
+
+  leftArg = demisto.args()["left"]
+  rightArg = demisto.args()["right"]
+
+  left = to_float(leftArg)
+  fromRange, toRange = parse_range(rightArg)
+
+  if INVALID_ARG in [left, fromRange,toRange]:
+      demisto.error("InRange - invalid arguments. left shuld be a number, right should be from,to (e.g. '1,8'). got left - %s, right - %s" % (leftArg, rightArg))
+      demisto.results(False)
+  else:
+      demisto.results(left >= fromRange and left <= toRange)
+
+  sys.exit(0)
+type: python
+tags:
+- filter
+- number
+comment: |-
+  checks if left side is in range of right side (from,to anotation)
+  e.g. - InRange left=4right=1,8 will return true
+enabled: true
+args:
+- name: left
+  required: true
+  description: value to check if in range (e.g. 4)
+- name: right
+  required: true
+  description: range to check against with the form of from,to (e.g. 2,8)
+scripttarget: 0
+runonce: false
```

Comment:
no need for this right?

Reply:
it imported automatically with the script
did we remove this feature?